openquake/hazardlib/tests/gsim/gmpe_tables_test.py

```diff
@@ -0,0 +1,753 @@
+# The Hazard Library
+# Copyright (C) 2015, GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import unittest
+import collections
+import mock
+
+import h5py
+import numpy as np
+from scipy.interpolate import interp1d
+
+from openquake.hazardlib import const
+from openquake.hazardlib.gsim.base import (
+    GMPE, IPE, SitesContext, RuptureContext, DistancesContext, GMPETable,
+    AmplificationTable, hdf_arrays_to_dict)
+from openquake.hazardlib.tests.gsim.utils import BaseGSIMTestCase
+from openquake.hazardlib import const
+from openquake.hazardlib import imt as imt_module
+
+
+BASE_DATA_PATH = os.path.join(os.path.dirname(__file__),
+                              "data",
+                              "GSIMTABLES")
+
+
+def midpoint(low, high, point=0.5):
+    """
+    Returns the logarithmic midpoint between two value
+    """
+    return 10.0 ** (point * (np.log10(low) + np.log10(high)))
+
+
+class HDFArraysToDictTestCase(unittest.TestCase):
+    """
+    Tests the conversion of a group containing a set of datasets(array) into
+    a dictionary
+    """
+    def setUp(self):
+        """
```

Comment:
Empty docstring; if there nothing interesting to say you can remove it. Actually in tests it is best to remove docstrings and use comments.